compiler/src/iree/compiler/Dialect/Flow/Transforms/test/form_scalar_dispatches.mlir

```diff
@@ -0,0 +1,161 @@
+// RUN: iree-opt --pass-pipeline="builtin.module(func.func(iree-flow-form-scalar-dispatches))" --split-input-file %s | FileCheck %s
+
+#map = affine_map<() -> ()>
+func.func @simpleDAG(
+    %arg0 : tensor<f32>, %arg1 : tensor<f32>, %arg2 : tensor<f32>, %arg3 : tensor<f32>)
+    -> (tensor<f32>, tensor<f32>) {
+  %0 = tensor.empty() : tensor<f32>
+  %1 = linalg.generic {indexing_maps = [#map, #map, #map], iterator_types = []}
```

Comment:
will be nice when we turn detensoring on and this pass can also pick up arith ops - shame that we have to keep all this linalg goo around for what's a couple of arith ops.